sympy/printing/latex.py

```diff
@@ -399,6 +402,12 @@ def convert(expr):
                 else:
                     args = expr.args
```

Comment:
`expr.args` is a tuple, which is immutable and doesn't support `del`. You have to convert it to a `list`.